indico/modules/events/abstracts/templates/display/submit_abstract.html

```diff
@@ -1,12 +1,8 @@
-{% from 'events/display/indico/_common.html' import render_users %}
 {% from 'forms/_form.html' import simple_form %}
 
-{% macro render_abstract_submit_form(form, event) %}
-    <div class="header">
-        <div class="header-row">
-            <h3>{%- trans %}Submit abstract{% endtrans -%}</h3>
-        </div>
+<div class="header">
+    <div class="header-row">
+        <h3>{%- trans %}Submit abstract{% endtrans -%}</h3>
```

Comment:
Well, if you want to put extra characters in your code that have absolutely 0 effect, sure thing :lollipop: